### PR TITLE
VC-4225: Empty wallet module implementation

### PR DESCRIPTION
**Pull request by Chuiko-GIT**
## Description

Closes: #XXXX

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: modules/registrar.go [hunk 1]
```diff
@@ -2,6 +2,10 @@ package modules
```
Comment (PavelVlasenkoGo) [inline modules/registrar.go]: этот файл точно должен называться именно так?

File: modules/registrar.go [hunk 2]
```diff
@@ -2,6 +2,10 @@ package modules
 
 import (
 	"fmt"
+	walletstypes "git.ooo.ua/vipcoin/chain/x/wallets/types"
+	"github.com/forbole/bdjuno/v2/modules/vipcoin/chain/wallets"
+	vipcoinwalletssource "github.com/forbole/bdjuno/v2/modules/vipcoin/chain/wallets/source"
+	remotevipcoinwalletsssource "github.com/forbole/bdjuno/v2/modules/vipcoin/chain/wallets/source/remote"
 	"os"
```
Comment (PavelVlasenkoGo) [inline modules/registrar.go:5]: импорты не отсортированы

File: modules/registrar.go (hunk 1 above)
Comment (Chuiko-GIT) [reply modules/registrar.go]: по дефолту так назывался)

**Comment by IamStubborN** [comment]
@alex-bodnar свою таску на ревью перевел на тебя, проверь пожалуйста

File: modules/vipcoin/chain/wallets/source/remote/source.go [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+func NewSource(source *remote.Source, walletClient walletstypes.QueryClient) *Source {
+	return &Source{
+		Source:  source,
+		wallets: walletClient,
+	}
+}
+
+// GetWallets implements bankkeeper.Source
+func (s Source) GetWallets(addresses []string, height int64) ([]*walletstypes.Wallet, error) {
+	//header := remote.GetHeightRequestHeader(height)
```
Comment (alex-bodnar) [inline modules/vipcoin/chain/wallets/source/remote/source.go]: Удали, или раскоментируй.

**Comment by alex-bodnar** [comment]
LGTM

**Review by alex-bodnar — APPROVED** [review]

**Comment by PavelVlasenkoGo** [comment]
LGTM

**Pull request merged**